master.yaml

```diff
@@ -201,6 +201,32 @@ Resources:
                 Subnets: !GetAtt VPC.Outputs.PrivateSubnets
                 EcrImage: 845828040396.dkr.ecr.us-west-2.amazonaws.com/production/civic-2018:latest
 
+    Civic2019Disaster:
+       Type: AWS::CloudFormation::Stack
+       Properties:
+           TemplateURL: https://s3-us-west-2.amazonaws.com/hacko-infrastructure-cfn/fargate-services/fargate-frontend.yaml
+           Parameters:
+                ProjectName: civic-2019-disaster
+                HealthCheckPathName: /2019-disaster-game/
+                Listener: !GetAtt ALB.Outputs.Listener
+                ListenerTls: !GetAtt ALB.Outputs.ListenerTls
+                ListenerRulePriority: 100
+                ListenerRuleTlsPriority: 101
+                ListenerRule2Priority: 102
+                Host: civicplatform.org
```

Comment:
The Host + Path values should also add up to a unique addressing scheme - e.g. if one Service occupies civicplatform.org/disaster*, no other team can occupy that namespace.

So here, in the case of this disaster game, this combo will directly collide and conflict with the existing CivicPlatformService service.  Not knowing anything of the context, as a strawman suggestion I would recommend something like a new A record in Route 53 such as disastergame.civicplatform.org, at which point we could use that for the Host entry, and we could leave Path at `/*` (since no other container apps will likely want to use any routes under that hostname).

One challenge here is that the current Fargate-Frontend template assumes we have two alternate hostnames to reach the same application container - but in this case we have no such legacy to battle with.  I haven't had time or need to explore how deal with a single hostname for the frontend containers.